src/transformers/trainer_pt_utils.py

```diff
@@ -161,6 +161,8 @@ def distributed_concat(tensor: Any, num_total_examples: Optional[int] = None) ->
             return type(tensor)(distributed_concat(t, num_total_examples) for t in tensor)
         output_tensors = [tensor.clone() for _ in range(dist.get_world_size())]
         output_tensors = [t if len(t.shape) > 0 else t[None] for t in output_tensors]
+        if len(tensor.shape) <= 0:
+            tensor = tensor[None]
```

Comment:
Maybe we should move this before the definition of `output_tensors` so we can remove current line 163?

Reply:
Done!